src/pages/docs/platform/auth/capabilities.mdx

```diff
@@ -0,0 +1,897 @@
+
+You can also choose whether to restrict the API key to only channels, only [queues](/docs/platform/integrations/queues), or to match a set of channel or queue names. If you've chosen to restrict the API key to *selected channels and queues*, you can use a comma separated list of resources the API key can access, making use of wildcards to provide access to areas of your app. It is worth noting an API key will provide the same permissions to all resources it has access to.
+
+To view the capabilities for an existing API key:
+
+1. Sign into your [Ably dashboard](https://ably.com/dashboard).
+2. Select the **API Keys** tab.
+3. Click the **Settings** button for the key you want to check the capabilities for.
+
+### Token capabilities <a id="capabilities-token"/>
+
+Ably Tokens and JWTs are issued from an existing API key and their capabilities can, at most, match the capabilities of the issuing API key.
+
+If an API key must be shared with a third party, then it is recommended that [the principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege) is considered, assigning only the capabilities needed by that third party. Thus, any Ably requests authenticated using that API key or Ably-compatible tokens associated with that API key, will be restricted to the capabilities assigned to the API key.
+
+Capabilities can be set when creating a token or token request, as shown in the following example:
+
+<Code>
+```javascript
+  var tokenParams = { clientId: 'foo', capability: JSON.stringify(capability) };
+  const tokenRequest = await ably.auth.createTokenRequest(tokenParams);
+```
+
+```python
+token_params = {
+    'client_id': 'foo',
+    'capability': json.dumps(capability)
+}
+
+token_request = await ably_rest.auth.create_token_request(token_params)
+```
+
+```php
+$tokenParams = [
+    'clientId' => 'client@example.com',
+    'capability' => json_encode($capability)
+];
+$tokenRequest = $rest->auth->createTokenRequest($tokenParams);
+```
+
+```go
+rest, _ := ably.NewREST(
+  ably.WithKey("xVLyHw.I2jW-g:dOTlhEt-nIubVAPMAUJnGv-_F8BZ7xNYnXdajpGaISg"))
+
+// Define the capability
+capability := map[string][]string{
+  "*": {"*"},
+}
+
+capabilityJSON, err := json.Marshal(capability)
+if err != nil {
+  log.Fatalf("Failed to marshal capability: %v", err)
+}
+
+// Define the token parameters
+tokenParams := &ably.TokenParams{
+  ClientID:   "foo",
+  Capability: string(capabilityJSON),
+}
+
+// Create a token request
+tokenRequest, err := rest.Auth.CreateTokenRequest(tokenParams)
+if err != nil {
+  log.Fatalf("Failed to create token request: %v", err)
+}
+```
+
+```flutter
+final tokenParams = ably.TokenParams(
+  clientId: 'foo',
+  capability: jsonEncode(capability),
+);
+
+final tokenRequest = await rest.auth.createTokenRequest(tokenParams: tokenParams);
+```
+</Code>
+
+#### Token capability determination <a id="capability-determination"/>
```

Comment:
It's not clear that there are 3 areas here without goin through all of the copy (and code). It would be nice to address that here in the section intro